### PR TITLE
7014 - Centered drilldown button within Datagrid cell.

### PR DESCRIPTION
**Pull request by RJ-Csc**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Adjusted positioning of elements within Datagrid cell to be centered. This is to fix the drilldown button not being vertically centered at medium row height.

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/7014

**Steps necessary to review your pull request (required)**:

- Pull this branch, build, and run the app.
- Go to: http://localhost:4000/components/datagrid/example-drilldown?theme=new&mode=dark&colors=0066D4
- Click the "More" button and select "Medium".
- Drilldown button should be centered.

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@RJ-Csc this is i think a risky change because you changed the defaults of the cells for every type of cell. If you look in the example it seems like he link is now not centered but the drill down button is https://main-enterprise.demo.design.infor.com/components/datagrid/example-drilldown

Can you find a different fix that only touchs the drill down button? (class: datagrid-drilldown)

**Review by tmcconechy — DISMISSED** [review]

File: src/components/datagrid/_datagrid-new.scss [hunk 1]
```diff
@@ -493,6 +497,10 @@
   .datagrid-trigger-cell .icon.icon-search-list {
     top: 4px;
   }
+
+  tbody tr td .datagrid-drilldown {
```
Comment (yohannahbautista) [inline src/components/datagrid/_datagrid-new.scss:501]: Very small, if you can put this up so it looks sorted, just to make it tidier
image

**Review by janahintal — APPROVED** [review]
Approving 
image

**Review by jbrcna — APPROVED** [review]

**Review by glenlieorillo — APPROVED** [review]
https://github.com/infor-design/enterprise/assets/102151352/715b3bde-2d13-4abd-b459-080ae3e36477

**Pull request merged**